src/rebar_core.erl

```diff
@@ -84,7 +84,7 @@ process_command(State, Command) ->
                     %% to have both $REBAR_PROFILE set and use 'as' in a command
                     case rebar_state:current_profiles(State) of
                         [default] ->
-                            do([{default, hd(TargetProviders)} | tl(TargetProviders)], State);
+                            do([hd(TargetProviders) | tl(TargetProviders)], State);
```

Comment:
Hah, woops, yea. But na, it isn't needed, at least not with the v1.3.0 version of providers.